shared/models/http/http-error-codes.ts

```diff
@@ -4,7 +4,7 @@
  *
  * WebDAV and other codes useless with regards to PeerTube are not listed.
  */
-export const enum HttpStatusCode {
+export enum HttpStatusCode {
```

Comment:
Please keep the `const` here that reduce client bundle size

I'm not sure the HTTP status label is very interesting. If you think so, you'll have to build the object yourself

Reply:
Yeah, that was the price for this feature. If you don't think it is worth it to mention the label, I revert that.

Reply:
Done